Octokit/Octokit.csproj

```diff
@@ -17,6 +17,8 @@
 		<PackageLicenseExpression>MIT</PackageLicenseExpression>
 		<PackageTags>GitHub API Octokit linqpad-samples dotnetcore</PackageTags>
 		<Copyright>Copyright GitHub 2017</Copyright>
+    <LangVersion>9</LangVersion>
```

Comment:
I'm a little hesitant about this change here. My thinking was that this was necessary in order to have a property like `User? SuspendedBy`, otherwise there is a compile-time error. This didn't introduce any new warnings or errors in unit tests. However, when referencing these changes locally, and testing from a consuming application's perspective, this seems to introduce breaking changes related to potentially nullable references. Essentially, I'd like to be able to use nullable reference types, but would love to avoid changing the project configuration like this.